### DISCUSSION TITLE
Tutorial for embedding HLS streams

### DISCUSSION THREAD

**Discussion by phil-hei** [mozilla/hubs#4495] [General Questions & Discussion] [answered]
Is there any tutorial available on how to embed HLS streams into Hubs? Or does Hubs support RTMP, e.g. from a Webex/Teams live stream?

**Comment by worph** [comment, marked as answer]
Hubs support HLS and RTMP. from my knowledge you just have to past the link in it. But I have never tested from a Teams live stream
